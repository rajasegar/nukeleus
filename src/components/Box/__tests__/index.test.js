import React from 'react';
import { renderWithTheme } from '../../../test-utils';

import { Box } from '../index';

describe('Box component test suite', () => {
  it('Should render Box', async () => {
    const { container } = renderWithTheme(<Box> box </Box>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
