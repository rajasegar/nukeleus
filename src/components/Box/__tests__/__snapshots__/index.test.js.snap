// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box component test suite Should render Box 1`] = `
.c0 .cursor-pointer {
  cursor: pointer;
}

<div
  class="c0"
>
   box 
</div>
`;
